stdlib/tkinter/__init__.pyi

```diff
@@ -1775,24 +1775,24 @@ class Checkbutton(Widget):
         # and list[int] are incompatible. Also, we would need a way to
         # specify "Checkbutton not associated with any variable", which is
         # done by setting variable to empty string (the default).
-        offvalue: Any = ...,
-        onvalue: Any = ...,
-        overrelief: _Relief | Literal[""] = ...,
-        padx: _ScreenUnits = ...,
-        pady: _ScreenUnits = ...,
-        relief: _Relief = ...,
+        offvalue: Any = 0,
+        onvalue: Any = 1,
+        overrelief: _Relief | Literal[""] = "",
+        padx: _ScreenUnits = 1,
+        pady: _ScreenUnits = 1,
+        relief: _Relief = "flat",
         selectcolor: str = ...,
-        selectimage: _ImageSpec = ...,
-        state: Literal["normal", "active", "disabled"] = ...,
-        takefocus: _TakeFocusValue = ...,
-        text: float | str = ...,
+        selectimage: _ImageSpec = "",
+        state: Literal["normal", "active", "disabled"] = "normal",
+        takefocus: _TakeFocusValue = "",
+        text: float | str = "",
         textvariable: Variable = ...,
-        tristateimage: _ImageSpec = ...,
-        tristatevalue: Any = ...,
-        underline: int = ...,
-        variable: Variable | Literal[""] = ...,
-        width: _ScreenUnits = ...,
-        wraplength: _ScreenUnits = ...,
+        tristateimage: _ImageSpec = "",
+        tristatevalue: Any = "",
+        underline: int = -1,
+        variable: Variable | Literal[""] = "",
```

Comment:
This doesn't seem correct. By default, there is a dummy variable with the same name as the widget itself:

```
>>> import tkinter
>>> tkinter.Checkbutton()['variable']
<parsedVarName object: '!checkbutton1'>
```

(I spotted this with a quick and dirty script that reads the diff of this PR, instantiates widgets, and grabs their default values.)